### PR TITLE
Fix requires firewalld version in comments for permanent parameter.

### PR DESCRIPTION
**Pull request by piotrtrojanowski-dev**
##### SUMMARY
Version of firewalled seems to be wrong. 3.0.9 version of that package does not exist.
The latest release of firewalled: https://github.com/firewalld/firewalld/releases

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
ansible.posix.firewalld

```

### CODE REVIEW

File: plugins/modules/firewalld.py [hunk 1]
```diff
@@ -81,7 +81,7 @@
   permanent:
     description:
       - Should this configuration be in the running firewalld configuration or persist across reboots.
-      - As of Ansible 2.3, permanent operations can operate on firewalld configs when it is not running (requires firewalld >= 3.0.9).
+      - As of Ansible 2.3, permanent operations can operate on firewalld configs when it is not running (requires firewalld >= 0.3.9).
```
Comment (saito-hideki) [inline plugins/modules/firewalld.py:84]: Thank you for fixing this. Looks good to me :)

**Review by saito-hideki — CHANGES_REQUESTED** [review]
Also, could you create a new changelog fragments file to describe this fix? You can add a new file like `213-fix-wrong-firewalld-version-info.yml` under https://github.com/piotrtrojanowski-dev/ansible.posix/tree/patch-2/changelogs/fragments/

And it should contain like below short description of this change:
```
---
trivial:
  - firewalld - fix wrong version number in the module document.
```

You can see more details here: https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment

Update information for the release of the new version is generated from this fragments file. So once you added the changelog fragments file, I'll approve this PR.

thanks! :)

**Comment by saito-hideki** [comment]
Closing and re-opening for CI trigger.

**Comment by saito-hideki** [comment]
Closing and re-opening for CI trigger.

**Review by saito-hideki — APPROVED** [review]
I have added changelog fragment file. Thanks!

**Comment by saito-hideki** [comment]
@Akasurde @Andersson007
If it is possible, can you guys review for this PR?

**Review by Andersson007 — APPROVED** [review]

**Pull request merged**